metaserver/src/main/scala/scala/meta/languageserver/ScalametaEnrichments.scala

```diff
@@ -36,8 +35,8 @@ object ScalametaEnrichments {
       case Defn.Macro(_)  => SymbolKind.Constructor
       case Defn.Class(_)  => SymbolKind.Class
       case Defn.Trait(_)  => SymbolKind.Interface
-      case Defn.Object(_) => SymbolKind.Namespace
-      case Pkg.Object(_)  => SymbolKind.Module
+      case Defn.Object(_) => SymbolKind.Module
+      case Pkg.Object(_)  => SymbolKind.Namespace
```

Comment:
Strictly speaking, package object are modules like regular objects. But having a different icon in the UI to distinguish package objects from objects makes sense.

Reply:
Yep, that's what I thought